management-gateway/src/main/java/org/frankframework/management/gateway/HazelcastOutboundGateway.java

```diff
@@ -170,6 +169,9 @@ public void afterPropertiesSet() throws Exception {
 		hzInstance = HazelcastConfig.newHazelcastInstance("console");
 		SpringUtils.registerSingleton(applicationContext, "hazelcastOutboundInstance", hzInstance);
 
+		IMap<String, String> config = hzInstance.getMap("frank-configuration");
+		config.set("jwks", jwtGenerator.getPublicJwkSet());
```

Comment:
'frank-configuration' en 'jwks' kan je wellicht als constanten opnemen in `HazelcastConfig` aangezien je ze hier en in de InboundGateway gebruikt